code/lib/codemod/src/transforms/__tests__/mdx-to-csf.test.ts

```diff
@@ -0,0 +1,412 @@
+import * as fs_ from 'node:fs';
+import { expect, test } from '@jest/globals';
+import dedent from 'ts-dedent';
+import jscodeshift, { nameToValidExport } from '../mdx-to-csf';
+
+expect.addSnapshotSerializer({
+  print: (val: any) => (typeof val === 'string' ? val : JSON.stringify(val, null, 2) ?? ''),
+  test: () => true,
+});
+
+jest.mock('node:fs');
+const fs = fs_ as jest.Mocked<typeof import('node:fs')>;
+
+beforeEach(() => {
+  fs.existsSync.mockImplementation(() => false);
+});
+
+test('rewrite import', () => {
+  const input = dedent`
+      import { Meta, Story } from '@storybook/addon-docs';
+    `;
+
+  const mdx = jscodeshift({ source: input, path: 'Foobar.stories.mdx' });
+
+  expect(mdx).toMatchInlineSnapshot(`
+    import { Meta, Story } from '@storybook/blocks';
+    import * as FoobarStories from './Foobar.stories';
```

Comment:
Is it weird that it adds this line when no stories were defined in the file?

Reply:
Right, changed that.